apps/minifront/src/state/ibc-out.ts

```diff
@@ -242,7 +243,6 @@ export const filterBalancesPerChain = (
   const assetIdsToCheck = [penumbraAssetId, ...assetsWithMatchingChannel];
 
   return allBalances.filter(({ balanceView }) => {
-    const metadata = getMetadata(balanceView);
-    return assetIdsToCheck.some(assetId => assetId.equals(metadata.penumbraAssetId));
+    return assetIdsToCheck.some(assetId => assetId.equals(getAssetIdFromValueView(balanceView)));
```

Comment:
Unknown assets also have an `assetId`, but no `metadata`. 

Theoretically, this code allows for an unknown asset to be sent via IBC. But in fact, we can never match an unknown asset with an asset from some ibc channel, because in this case the asset is available in the registry and cannot be an unknown asset